cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/Changes.java

```diff
@@ -35,7 +35,7 @@
 
     private final List<DocumentRevision> results;
 
-    protected Changes(long lastSequence, List<DocumentRevision> results) {
+    public Changes(long lastSequence, List<DocumentRevision> results) {
```

Comment:
Perhaps add a comment indicating that API users are not expected to construct this and should call `com.cloudant.sync.datastore.Datastore#changes` since it is now publicly visible.